modules/vass/app/services/vass/redis_client.rb

```diff
@@ -25,6 +25,29 @@ def initialize
       @settings = Settings.vass
     end
 
+    # ------------ Settings Accessors ------------
+    # Note: Using explicit methods instead of delegate to avoid Ruby 3.3 warnings
+    # about forwarding to private OpenStruct methods
+
+    ##
+    # Returns the OTC expiry time from settings.
+    #
+    # @return [ActiveSupport::Duration] OTC expiry duration
+    #
+    # rubocop:disable Rails/Delegate
+    def redis_token_expiry
+      @settings.redis_token_expiry
+    end
+
+    def redis_otc_expiry
+      @settings.redis_otc_expiry
+    end
+
+    def redis_session_expiry
+      @settings.redis_session_expiry
+    end
+    # rubocop:enable Rails/Delegate
```

Comment:
I must have missed this in my review earlier. Is there any way you can not disable the rubocop here?